docs/hub/security.md

```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
 ```
 
 Once pushed on the Hub, you should see the commit with a "Verified" badge.
+
+## Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
+
+Scanning is triggered at each commit or when you visit a repository page.
+
+Here is an [example view](https://huggingface.co/mcpotato/42-eicar-street/tree/main) of an infected file :
+
+![/docs/assets/hub/eicar-hub-tree-view.png](/docs/assets/hub/eicar-hub-tree-view.png)
+
+![/docs/assets/hub/eicar-hub-file-view.png](/docs/assets/hub/eicar-hub-file-view.png)
+
+_Note_: if your file has neither an ok or infected badge, it could mean that it is either currently being scanned / waiting to be scanned or that there was an error during the scan. It can take up to a few minutes to be scanned.
```

Comment:
Unrelated to this PR, but should we also show a big warning in https://huggingface.co/mcpotato/42-eicar-street when a repo has infected files?

Reply:
It's kind of there already, just on the files tab

Reply:
I don't think it's the same. As a user of a model, I would likely not open the files tab, just read the model card and even click the "how to clone" button in the model card to download the model. If a model is scanned to be unsafe, we should show it more prominently imo. (happy to follow up discussing this in slack :D)

Reply:
I've just made it so that calls to my API take on avg 5-10 ms. Is that too much of an overhead ?

Reply:
no 5-10ms is fine